src/redist/targets/GenerateLayout.targets

```diff
@@ -183,6 +183,20 @@
         <RelativeLayoutPath>packs/%(PackageName)/%(PackageVersion)</RelativeLayoutPath>
       </BundledLayoutPackage>
 
+      <BundledLayoutPackage Include="MicrosoftNetCoreAppRuntimePackNupkg" Condition="'$(BundleRuntimePacks)' == 'true'">
+        <PackageName>Microsoft.NETCore.App.Runtime.$(SharedFrameworkRid)</PackageName>
```

Comment:
Is `SharedFrameworkRid` the same as `ProductMonikerRid`?  `ProductMonikerRid` is used earlier for the known framework reference RIDs.

Reply:
I'm using the same parameters as we were are already using for the app host, and they are both equal to the non-portable rid.